scripts/validate-code.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+sbt ++$TRAVIS_SCALA_VERSION validateCode
```

Comment:
Need to handle this failing

```suggestion
sbt ++$TRAVIS_SCALA_VERSION validateCode || travis_terminate 1
```